### ISSUE TITLE
Cannot install due to missing yiisoft files

### ISSUE THREAD

**Issue by benedetto** [humhub/humhub#1754]
I am trying to install humhub according to the [installation guide](https://www.humhub.org/docs/guide-admin-installation.html):
I have cloned the stable branch from github (```git clone```, ```git checkout stable```) and fetched additional files via composer as described in the guide. However, when I open the url where the humhub installation is located, instead of the expected installer pages, I only get an empty page in my browser.

If I instead call the index.php directly from the command line on the server via ```php index.php```, I get the following error messages:

```
PHP Warning:  require(/var/www/.../humhub/protected/vendor/yiisoft/yii2/Yii.php): failed to open stream: No such file or directory in /var/www/.../humhub/index.php on line 14
PHP Fatal error:  require(): Failed opening required '/var/www/.../humhub/protected/vendor/yiisoft/yii2/Yii.php' (include_path='.:/package/host/localhost/php-5.6.21/lib/php') in /var/www/.../humhub/index.php on line 14
```

So obviously there are some files missing in the ```protected/vendor``` folder, however, those are also not present in the github repository. What am I missing/doing wrong? I would be grateful for any hint.

### Versions:
PHP Version: 5.6.21
Humhub version: stable branch from Github
OS: CentOS 5 Linux
Web Server: Apache 2.2

**Comment by cebe** [comment]
are you sure composer install process completed without errors?

**Comment by benedetto** [comment]
I think so. This is the output if I run ```php composer.phar global require "fxp/composer-asset-plugin:~1.1.1"  ```:

```
Changed current directory to /home/<username>/.composer
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing fxp/composer-asset-plugin (v1.1.4)
    Downloading: 100%         

Writing lock file
Generating autoload files
```

**Comment by buddha87** [comment]
You have to execute `composer update` in your humhub root directory in order to load the dependencies.

**Comment by benedetto** [comment]
Ahhhh, there comes the light. I remember that I had tried this command once and it had thrown a "command not found" error but I since this had happened somewhere in the middle of my various attempts to get all this running, I had forgotten about it.

Now, upon reading a bit more about composer, I found out that the ```composer update``` command does only work if composer has been installed on your system and the ```composer``` command is available as an executable command. In my case I only had chosen the approach of putting it locally into my humhub directory. In this case there is *no* composer command, instead the respective commands can be called by ```php composer.phar <parameter>```, so in my case ```php composer.phar update``` did the trick. Maybe the installation guide should contain some hint about that.
